### ISSUE TITLE
Auto-complete - Partial matches

### ISSUE THREAD

**Issue by LunarMarathon** [LunarMarathon/graphToTable#14]
When the input is more than one word, the autocomplete doesn't show the suggestion when words other than the first word is typed ex: 
![image](https://user-images.githubusercontent.com/113847439/195345515-1036bdcf-003a-4606-a994-8af86d94df73.png)
But myt- doesn't get Greek mythology suggestion 
![image](https://user-images.githubusercontent.com/113847439/195345622-cc399609-28e8-4d96-9123-1416e8bc9be4.png)

**Comment by singharyan1007** [comment]
Hi @LunarMarathon I have an idea about this issue. Can you assign it to me?

**Comment by LunarMarathon** [comment]
Yep, I guess you can split the input and use the same logic, try to include words separated by hyphen too if you can (ex: multi-layer). I'll assign it to you
